transportation.sk

```diff
@@ -122,14 +122,19 @@ the wild update:
 
 	# boat with chest
 	spruce (chest boat¦s|boat¦s with chest) = minecraft:spruce_chest_boat[relatedEntity=spruce chest boat]
+	spruce boat¦s = spruce boat, spruce chest boat
 	birch (chest boat¦s|boat¦s with chest) = minecraft:birch_chest_boat[relatedEntity=birch chest boat]
+	birch boat¦s = birch boat, birch chest boat
 	jungle (chest boat¦s|boat¦s with chest) = minecraft:jungle_chest_boat[relatedEntity=jungle chest boat]
+	jungle boat¦s = jungle boat, jungle chest boat
 	acacia (chest boat¦s|boat¦s with chest) = minecraft:acacia_chest_boat[relatedEntity=acacia chest boat]
+	acacia boat¦s = acacia boat, acacia chest boat
 	oak (chest boat¦s|boat¦s with chest) = minecraft:oak_chest_boat[relatedEntity=oak chest boat]
+	oak boat¦s = oak boat, oak chest boat
 	dark oak (chest boat¦s|boat¦s with chest) = minecraft:dark_oak_chest_boat[relatedEntity=dark oak chest boat]
+	dark oak boat¦s = dark oak boat, dark oak chest boat

```

Comment:
I don't think these changes can work. It makes `give the player a spruce boat` have the chance to give the player a spruce boat with a chest and leaves no way of specifically giving a non-chest boat. If we want to include this, it should be like `any <wood> boat¦s`